lib/iris/tests/unit/coord_categorisation/test_add_categorised_coord.py

```diff
@@ -116,6 +116,11 @@ def make_cube(self, calendar):
 
     def test_calendars(self):
         for calendar in calendars:
+            # Skip the Julian calendar due to
+            # https://github.com/Unidata/netcdftime/issues/13
+            # Remove this if block once the issue is resolved.
```

Comment:
Is this issue present as a result in changing versions of netcdftime?

If so, would pinning the version of netcdftime be the preferred option until https://github.com/Unidata/netcdftime/issues/13 is resolved?